tls/s2n_recv.c

```diff
@@ -204,8 +204,8 @@ ssize_t s2n_recv_impl(struct s2n_connection *conn, void *buf, ssize_t size, s2n_
             conn->in_status = ENCRYPTED;
             continue;
         }
-
-        out.size = MIN(size, s2n_stuffer_data_available(&conn->in));
+        POSIX_ENSURE(size >= 0, S2N_ERR_SAFETY);
```

Comment:
nit
```suggestion
        RESULT_ENSURE_GTE(size, 0);
```

Reply:
This errors with incompatible result type.

Reply:
sorry - suggestion should say POSIX_ENSURE_GTE